Hakawai/Mentions/HKWMentionsStartDetectionStateMachine.m

```diff
@@ -112,37 +123,14 @@ - (void)validStringInserted:(NSString *)string
     }
 }
 
-- (void)characterTyped:(unichar)c asInsertedCharacter:(BOOL)inserted previousCharacter:(unichar)previousCharacter {
+- (void)characterTyped:(unichar)c
+   asInsertedCharacter:(BOOL)inserted
+     previousCharacter:(unichar)previousCharacter
+            nextString:(nullable NSString *)nextString {
```

Comment:
can you call this 

wordFollowingTypedCharacter:

and document this function?

Reply:
Renamed `nextString` to `wordFollowingTypedCharacter` and documented the function.